src/archanaNJun20/MissingInteger.java

```diff
@@ -0,0 +1,19 @@
+package archanaNJun20;
+
+public class MissingInteger {
+	void printMissingInteger(int array[],int n){
+		int total=((n*(n+1))/2);
+		int sum=0;
+		for(int index=0;index<array.length;index++){
+			sum=sum+array[index];
+		}
+		int differnce=total-sum;
+		System.out.println("Missing number in array is: "+differnce);
```

Comment:
You can directly write total-sum; in syso statement.

Reply:
The code is updated based on comments.